src/app/pages/ArticlePage/ArticleHeadline/index.tsx

```diff
@@ -0,0 +1,54 @@
+/** @jsx jsx */
+/* @jsxFrag React.Fragment */
+import { jsx } from '@emotion/react';
+import React, { useContext } from 'react';
+import { RequestContext } from '#contexts/RequestContext';
+import useViewTracker from '#app/hooks/useViewTracker';
+import useToggle from '#hooks/useToggle';
+import CallToActionLinkWithChevron from '#app/components/CallToActionLinkWithChevron';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Headings from '#containers/Headings';
+import { ComponentToRenderProps } from '../types';
+import styles from './index.styles';
+
+const ArticleHeadline = (props: ComponentToRenderProps) => {
+  const { pathname, isLite } = useContext(RequestContext);
+  const { translations } = useContext(ServiceContext);
+  const eventTrackingData = { componentName: 'canonical-lite-cta' };
+  const { enabled: showCTA } = useToggle('liteSiteCTA');
+  const viewRef = useViewTracker(eventTrackingData);
+
+  const articleDataSavingLinkText =
+    translations?.liteSite?.articleDataSavingLinkText ?? 'Data-saving Version';
```

Comment:
Feedback from demo: update text
```suggestion
    translations?.liteSite?.articleDataSavingLinkText ?? 'Data saving version';
```